### PR TITLE
Update the contents of NO2 and GRDI content pages.

### PR DESCRIPTION
**Pull request by SwordSaintLancelot**
## What am I changing and why
Dataset overview content updates for
1. GRDI - from https://docs.google.com/document/d/1q0EDmAUf5FU6KN0Y4k1YkC3XSYEqLblsVmDJ3BZMVxU/edit
2. NO2 - from https://docs.google.com/document/d/1YKNUXSbd_2HOt6jHGSFHSh4WZB2IYk_ffFFh2WZH9dU/edit

Note: This content has been sent for review to the respective data providers and may require minor updates after the review.

### CODE REVIEW

File: datasets/grdi-v1.data.mdx [hunk 1]
```diff
@@ -4,23 +4,26 @@ name: "The Global Gridded Relative Deprivation Index, Version 1"
 description: This dataset characterizes the relative levels of multidimensional deprivation and poverty based on sociodemographic and satellite data
 media:
   src: ::file ./grdi--dataset-cover.jpg
-  alt: Shacks along a river almost collapsing
+  alt: Figure by Jordan Opel
```
Comment (slesaad) [inline datasets/grdi-v1.data.mdx:7]: revert this to what it was

>An alt tag, also known as "alt attribute" and "alt description," is an HTML attribute applied to image tags to provide a text alternative for search engines.

Comment (slesaad) [reply datasets/grdi-v1.data.mdx:7]: The "Figure by Jordan Opel" comes from the author name below

File: datasets/grdi-v1.data.mdx [hunk 2]
```diff
@@ -195,111 +198,49 @@ layers:
...
+        - "#3b528b"
+        - "#21918c"
+        - "#5ec962"
+        - "#fde725"
 ---
 
-<Block>
+<Block type='wide'>
   <Prose>
     ## About
```
Comment (slesaad) [inline datasets/grdi-v1.data.mdx:210]: remove About to keep it consistent across datasets

File: datasets/no2.data.mdx [hunk 3]
```diff
@@ -1,32 +1,37 @@
 ---
 id: no2
-name: 'Nitrogen Dioxide'
+name: "Recent Changes in Atmospheric Nitrogen Dioxide"
 featured: true
```
Comment (slesaad) [inline datasets/no2.data.mdx:4]: remove featured

File: datasets/no2.data.mdx [hunk 4]
```diff
@@ -1,32 +1,37 @@
...
-    title: 'Interactive session in VEDA 2i2c JupyterHub (requires account)'
+    title: "Interactive session in VEDA 2i2c JupyterHub (requires account)"
 media:
   src: ::file ./no2--dataset-cover.jpg
-  alt: Power plant shooting steam at the sky.
+  alt: Figure by Mick Truyts.
   author:
     name: Mick Truyts
-    url: https://unsplash.com/photos/x6WQeNYJC1w
+    url: https://unsplash.com/photos/sea-waves-crashing-on-shore-under-white-clouds-x6WQeNYJC1w
```
Comment (slesaad) [inline datasets/no2.data.mdx:18]: remove usage

File: datasets/no2.data.mdx [hunk 5]
```diff
@@ -1,32 +1,37 @@
...
+    values:
+      - NASA
+  - name: Product Type
+    values:
+      - Satellite Observations
 layers:
   - id: no2-monthly
     stacCol: no2-monthly
-    name: No2
+    name: Nitrogen Dioxide (NO2)
```
Comment (slesaad) [inline datasets/no2.data.mdx:32]: Replace all NO2 -> NO₂

File: datasets/no2.data.mdx (hunk 4 above)
Comment (SwordSaintLancelot) [reply datasets/no2.data.mdx:18]: Is usage the double quotes?

File: datasets/no2.data.mdx [hunk 6]
```diff
@@ -1,21 +1,14 @@
...
+name: 'Recent Changes in Atmospheric Nitrogen Dioxide'
+description: 'Since the outbreak of the novel coronavirus, atmospheric concentrations of nitrogen dioxide have changed by as much as 60% in some regions.'
 media:
   src: ::file ./no2--dataset-cover.jpg
-  alt: Figure by Mick Truyts.
+  alt: Power plant shooting steam at the sky.
   author:
     name: Mick Truyts
     url: https://unsplash.com/photos/sea-waves-crashing-on-shore-under-white-clouds-x6WQeNYJC1w
+    alt: Figure by Mick Truyts
```
Comment (slesaad) [inline datasets/no2.data.mdx:11]: remove

File: datasets/no2.data.mdx [hunk 7]
```diff
@@ -47,19 +44,21 @@ layers:
         }
     legend:
       type: gradient
-      min: "Less"
-      max: "More"
+      min: 'Less'
+      max: 'More'
+      unit:
+        label: Molecules NO<sub>2</sub>/cm2
```
Comment (slesaad) [inline datasets/no2.data.mdx:50]: superscript 2 in cm2

File: datasets/no2.data.mdx [hunk 8]
```diff
@@ -79,116 +78,53 @@ layers:
...
-        - 30E14
-    legend:
+      min: '< -3'
+      max: '> 3'
       unit:
-        label: mol/cm2
-      type: gradient
-      min: 0
-      max: 30e14
+        label: Molecules NO<sub>2</sub>/cm2
```
Comment (slesaad) [inline datasets/no2.data.mdx:84]: same here

File: datasets/no2.data.mdx [hunk 9]
```diff
@@ -79,116 +78,53 @@ layers:
...
-## Interpreting the data
-Nitrogen dioxide has a relatively short lifespan in the atmosphere. Once it is emitted, it lasts only a few hours before it dissipates, so it does not travel far from its source.
+  <Prose>
+    ## Source Data Access
+    The monthly tropospheric NO<sub>2</sub> data can be accessed here:
+    [https://avdc.gsfc.nasa.gov/pub/data/satellite/Aura/OMI/V03/L3/OMNO2d_HR/OMNO2d_HRM/](https://avdc.gsfc.nasa.gov/pub/data/satellite/Aura/OMI/V03/L3/OMNO2d_HR/OMNO2d_HRM/)
 
-Because nitrogen dioxide is primarily emitted from burning fossil fuels, changes in its atmospheric concentration can be related to changes in human activity, if the data are properly processed and interpreted.
+          ## Acknowledgment
+          Acknowledgement of the teams/organizations that contributed to the product. This can often be found in dataset documentation from source
```
Comment (slesaad) [inline datasets/no2.data.mdx:119]: 👀

Comment (SwordSaintLancelot) [reply datasets/no2.data.mdx:119]: I don't know what to add here. It is written in the document. I commented in the google doc, who is suppose to write it.

**Review by slesaad — CHANGES_REQUESTED** [review]
Comment out the acknowledgement section for now

**Comment by slesaad** [comment]
Also use the wide prose block so the data summary appears at the top right, remove any title like "Description" or "About"

check this PR - https://github.com/NASA-IMPACT/veda-config-eic/pull/39

File: datasets/no2.data.mdx [hunk 10]
```diff
@@ -47,7 +47,7 @@ layers:
       min: 'Less'
       max: 'More'
       unit:
-        label: Molecules NO<sub>2</sub>/cm2
+        label: Molecules NO<sub>2</sub>/cm<sub>2</sub>
```
Comment (slesaad) [inline datasets/no2.data.mdx:50]: superscript not subscript

File: datasets/no2.data.mdx [hunk 11]
```diff
@@ -81,7 +81,7 @@ layers:
       min: '< -3'
       max: '> 3'
       unit:
-        label: Molecules NO<sub>2</sub>/cm2
+        label: Molecules NO<sub>2</sub>/cm<sub>2</sub>
```
Comment (slesaad) [inline datasets/no2.data.mdx:84]: .

File: datasets/no2.data.mdx [hunk 12]
```diff
@@ -103,7 +103,7 @@ layers:
     - **Temporal Resolution:** Monthly
     - **Spatial Extent:** Global
     - **Spatial Resolution:** 0.1 degrees x 0.1 degrees
-    - **Data Units:** Molecules of nitrogen dioxide per square centimeter (Molecules NO<sub>2</sub>/cm2)
+    - **Data Units:** Molecules of nitrogen dioxide per square centimeter (Molecules NO<sub>2</sub>/cm<sub>2</sub>)
```
Comment (slesaad) [inline datasets/no2.data.mdx:106]: .

**Comment by slesaad** [comment]
Screenshot 2023-11-21 at 11 57 27 AM
these tags don't work in legends so can you fix those?

File: datasets/no2.data.mdx [hunk 13]
```diff
@@ -47,7 +47,7 @@ layers:
       min: 'Less'
       max: 'More'
       unit:
-        label: Molecules NO<sub>2</sub>/cm<sup>2</sup>
+        label: Molecules NO₂/cm2
```
Comment (slesaad) [inline datasets/no2.data.mdx:50]:
```suggestion
        label: Molecules NO₂/cm²
```

File: datasets/no2.data.mdx [hunk 14]
```diff
@@ -81,7 +81,7 @@ layers:
       min: '< -3'
       max: '> 3'
       unit:
-        label: Molecules NO<sub>2</sub>/cm<sup>2</sup>
+        label: Molecules NO₂/cm2
```
Comment (slesaad) [inline datasets/no2.data.mdx:84]:
```suggestion
        label: Molecules NO₂/cm²
```

**Review by slesaad — APPROVED** [review]

**Pull request merged**